lib/libc/glibc/sysdeps/loongarch/nptl/bits/pthreaddef.h

```diff
@@ -0,0 +1,32 @@
+/* pthread machine parameter definitions.
```

Comment:
Oh, I didn't realize LoongArch doesn't have a `pthreadtypes-arch.h`. I don't think we need this header then; they're not the same. So the only header we were actually missing was `bits/endianness.h`. So I guess just remove this one after all, and 68451422d16268cd6acc1b629ce60fe4a3a9d7c7 won't be needed then either.

Sorry for the confusion. :slightly_smiling_face:

Reply:
Removed. Is there any specific rule to tell whether a arch-specific header file is needed?

Reply:
Not that I'm aware of. Like I mentioned above, I'm not even sure *why* we duplicate these headers in `lib/libc/glibc/...` when they (usually) also end up in `lib/libc/include/...`. We should probably clean this up at some point.

For now, I just recommended duplicating the ones that were duplicated for RISC-V and also exist for LoongArch, which... just comes down to `bits/endianness.h`, actually.